### PR TITLE
Organize Evas-related dependencies

### PR DESCRIPTION
**Pull request by swift-kim**

### CODE REVIEW

File: shell/platform/tizen/channels/text_input_channel.cc [hunk 1]
```diff
@@ -5,7 +5,6 @@
 #include "text_input_channel.h"
 
 #include <Ecore.h>
-#include <Ecore_IMF_Evas.h>
```
Comment (bwikbs) [inline shell/platform/tizen/channels/text_input_channel.cc:8]: We don't need this anymore?

**Review by bwikbs — APPROVED** [review]
:+1:

File: shell/platform/tizen/channels/text_input_channel.cc (hunk 1 above)
Comment (swift-kim) [reply shell/platform/tizen/channels/text_input_channel.cc:8]: It seems.

cc @bbrto21

Comment (bbrto21) [reply shell/platform/tizen/channels/text_input_channel.cc:8]: This seems to have been included by mistake in the first place.

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**